apps/GroundControl/groundControl.py

```diff
@@ -0,0 +1,13 @@
+#Imports needed
+
+class GroundControl():
+	#Used to control the ground vehicle
+
+	def __init__(self):
+		# TODO: Create and Calibrate Motor Controllers
+		self.frontLeft = 0
```

Comment:
Soon there will be some more clear formatting rules for `obc`, but best to keep these variable names snake cased. That seems to be the convention we're following in the other applications, and is the generally accepted python convention for variable naming :).